Controller/SuggestionProviderController.php

```diff
@@ -34,6 +35,49 @@ class SuggestionProviderController extends ExceptionController
     protected $suggestionProviders = array();
 
     /**
+     * A set of configuration for the error handling.
+     *
+     * Contains the list of templates defined in the error section of the
+     * cmf_seo configuration.
+     *
+     * Looks like
+     *
+     * array(
+     *  'templates' => array(
```

Comment:
cause i thought there will be more than one configuration later. I think
in Kanban its called `muhda`.